bi_sql_editor/models/bi_sql_view_field.py

```diff
@@ -237,8 +237,8 @@ def _prepare_search_filter_field(self):
         self.ensure_one()
         res = ""
         if self.field_description and self.is_group_by:
-            res = """<filter name="%s" string="%s"
-                        context="{'group_by':'%s'}"/>""" % (
+            res = """<filter name="{}" string="{}"
+                        context="{{'group_by':'{}'}}"/>""".format(
```

Comment:
This change should not be in this PR ?

Reply:
This was changed automatically by pre-commit, I can remove it if you would like ?

Reply:
Ah ok, it must be pyupgrade that does it. Strange that it changes in this PR and while the main branch is green ?